core/deployment/src/main/java/io/quarkus/deployment/builditem/NativeEnableAllTimeZonesBuildItem.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class NativeEnableAllTimeZonesBuildItem extends MultiBuildItem {
```

Comment:
I think I would name it `NativeImageEnableAllTimeZonesBuildItem` and also rename the other one (and maybe deprecate the old one: we have a nice mechanism to deal with these deprecations, see `DeprecatedBuildItemProcessor`).

Saying so because all the other native image related build items are prefixed with `NativeImage` so better be consistent.

Reply:
done